va/va.go

```diff
@@ -320,30 +320,30 @@ func detailedError(err error) *probs.ProblemDetails {
 		if fmt.Sprintf("%T", netOpErr.Err) == "tls.alert" {
 			// All the tls.alert error strings are reasonable to hand back to a
```

Comment:
I think this comment as long since served its purpose and could be removed.

Reply:
Heh, reasonable, but I'd prefer not to do that in this PR, since all I'm changing here is the name of the error type.